backend-shared/config/db/deploymenttoapplicationmapping_test.go

```diff
@@ -0,0 +1,131 @@
+package db
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateandDeleteDeploymentToApplicationMapping(t *testing.T) {
+
+	testSetup(t)
+	defer testTeardown(t)
+
+	ctx := context.Background()
+	dbq, err := NewUnsafePostgresDBQueries(true, true)
+	if !assert.NoError(t, err) {
+		return
+	}
+	defer dbq.CloseDatabase()
+
+	_, managedEnvironment, _, gitopsEngineInstance, clusterAccess, err := createSampleData(t, dbq)
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	application := &Application{
+		Application_id:          "test-my-application",
+		Name:                    "my-application",
+		Spec_field:              "{}",
+		Engine_instance_inst_id: gitopsEngineInstance.Gitopsengineinstance_id,
+		Managed_environment_id:  managedEnvironment.Managedenvironment_id,
+	}
+
+	err = dbq.CheckedCreateApplication(ctx, application, clusterAccess.Clusteraccess_user_id)
+
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	deploymentToApplicationMapping := &DeploymentToApplicationMapping{
+		Deploymenttoapplicationmapping_uid_id: generateUuid(),
```

Comment:
Can you change this to `"test-"+generateUuid()`? That way we can call testSetup to remove them all.